blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/LogResource.java

```diff
@@ -44,16 +44,9 @@ private void writeLog(StaplerRequest req, StaplerResponse rsp, AcceptHeader acce
                 rsp.setHeader("Content-Disposition", "attachment; filename=log.txt");
             }
 
-            switch (accept.select("text/plain","text/html")) {
-                case "text/html":
-                    rsp.setContentType("text/html;charset=UTF-8");
-                    rsp.setStatus(HttpServletResponse.SC_OK);
-                    req.setAttribute("html", Boolean.valueOf(true));
-                    break;
-                case "text/plain":
-                    rsp.setContentType("text/plain;charset=UTF-8");
-                    rsp.setStatus(HttpServletResponse.SC_OK);
-            }
+            rsp.setContentType("text/plain;charset=UTF-8");
```

Comment:
Its the same code for both, we need to make sure steps are fine with it being text/plain